resources/views/dashboard/dashboard.blade.php

```diff
@@ -20,6 +20,12 @@
         </div>
         <hr>
     @endif
+    @if($liveEventTitle)
+        <div class="alert alert-warning text-center">
+            <strong><a href="/dashboard/controllers/live" alt="{{ $liveEventTitle }} Information">{{ $liveEventTitle }} Information</a></strong>
+        </div>
+        <hr>
+    @endif
```

Comment:
Should we only display here if it's published as well?

Reply:
Updated to make the variable single use and call the static accessor instead